### DISCUSSION TITLE
Pair Programming

### DISCUSSION THREAD

**Discussion by edchanonline** [earthlab-education/Earth-Analytics-2023-01-Intro#102] [Q&A01 Online Portfolio]
@earthlab-education/ea-students If anyone struggles with Git or GitHub Pages, happy to pair program to get your through!

**Comment by ellenalamont17** [comment]
Hi Ed, I may be inclined to take you up on this offer as we move forward. I seem to struggle with even the basics some times, so I might be nice to have a partner to bounce ideas off of. Looking forward to more to come!

**Reply by edchanonline** [reply]
Happy to! 💪 
Let me know
